### PR TITLE
Calc_dihedral docstring updated

### LINKED ISSUE #3936: update docs for calc_dihedral to indicate output range
The docs for dihedral calculations do not state the range of angles that are returned.

Based on the use of [atan2()](https://linux.die.net/man/3/atan2) from the C standard lib in
https://github.com/MDAnalysis/mdanalysis/blob/497f45e23c834780f9ca172b3e4d4ad7602136bc/package/MDAnalysis/lib/include/calc_distances.h#L761 the answer should be [–π, π].

### PR DESCRIPTION
**Pull request by Hakarishirenai**
Fixes #3936

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

File: package/MDAnalysis/lib/distances.py [hunk 1]
```diff
@@ -1636,7 +1636,7 @@ def calc_dihedrals(coords1: Union[np.ndarray, 'AtomGroup'],
         Array containing the dihedral angles formed by each quadruplet of
         coordinates. Values are returned in radians (rad). If four single
         coordinates were supplied, the dihedral angle is returned as a single
-        number instead of an array.
+        number instead of an array. Range of the dihedral angle is (:math:`-\pi`, :math:`\pi`)
```
Comment (orbeckst) [inline package/MDAnalysis/lib/distances.py]: Please stay with in 79 characters linewidth.

You can also make the whole range a math expression
```rest
:math:`(-\pi, \pi)`
```

Comment (orbeckst) [reply package/MDAnalysis/lib/distances.py]: Add an article 
```reST
The range of ...
```

**Review by orbeckst — CHANGES_REQUESTED** [review]
Thank you. Please address my comments.

Please also
- add yourself to AUTHORS
- add your GitHub handle to CHANGELOG for 2.4.0 at the end of https://github.com/MDAnalysis/mdanalysis/blob/3712439b418f2910ebac9233c5b5cdc0732bead0/package/CHANGELOG#L16-L17 (we don't need a CHANGELOG entry for a simple doc fix but you should be listed as contributing to the next release)

**Comment by Hakarishirenai** [comment]
Thank you for the help, have made necessary changes

**Comment by orbeckst** [comment]
Thank you @Hakarishirenai , I'll have a look again when the CI check have run. But it's looking very good at first glance.

As a new contributor, please also introduce yourself on the developer mailing list (see https://www.mdanalysis.org/#participating for the link) and mention your work  on this PR. It's important that we have an email address where we can reach you because of our ongoing effort to [change the license of MDAnalysis](https://www.mdanalysis.org/2022/11/07/relicensing/#relicensing-mdanalysis). We will not merge this PR until we know how we can contact you in the future.

**Comment by orbeckst** [comment]
Thank you for introducing yourself https://groups.google.com/g/mdnalysis-devel/c/CjIST5IkZyM/m/Xtcn3D_rAAAJ . I'll merge.

**Review by orbeckst — APPROVED** [review]

**Comment by orbeckst** [comment]
Thank you for your first commit! 🚀

**Pull request merged**